dev-support/cicd/README.md

```diff
@@ -34,3 +41,18 @@ Enter Menu ID:1
 ==== Merge PR Begin ====
 Enter Your Apache JIRA User name:
 ```
+
+## Manual mode
+
+### The Procedure of Merging PR
+
```

Comment:
add "install python 2.7.13 and pip insalll jira"?

Reply:
Done.